lib/orderbook/OrderBook.ts

```diff
@@ -620,11 +620,13 @@ class OrderBook extends EventEmitter {
   /**
    * Removes all or part of an order from the order book by its local id. Throws an error if the
    * specified pairId is not supported or if the order to cancel could not be found.
+   * @param allowEventualRemoval whether to allow an eventual async removal of the order in case
```

Comment:
Maybe specify here that an error is thrown if this is `false` but the order is on hold. Also it may be clearer to say `allowAsyncRemoval`.